src/components/ButtonSec.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export default function ButtonSec() {
```

Comment:
this component is almost the same as previous one, use properties to allow button customization